demo-shell-ng2/package.json

```diff
@@ -1,66 +1,28 @@
 {
-  "name": "Alfresco-Angular2-Demo",
-  "description": "Demo shell for Alfresco Angular2 components",
-  "version": "1.9.0",
-  "author": "Alfresco Software, Ltd.",
+  "name": "myapp",
+  "version": "0.0.0",
+  "license": "MIT",
   "scripts": {
-    "build": "npm run server-versions && rimraf dist && npm run webpack -- --config  config/webpack.prod.js --progress --profile --bail",
-    "build:dev": "npm run server-versions && rimraf dist && npm run style:dev && npm run webpack -- --config  config/webpack.dev.js --progress --profile --bail",
-    "style:dev": "npm run webpack -- --config  config/webpack.style.js --progress --profile --bail",
-    "start:dist": "npm run server-versions && wsrv -s dist/ -p 3000 -a 0.0.0.0",
-    "start": "npm run server-versions && npm run webpack-dev-server -- --config config/webpack.prod.js --progress  --content-base app/ ",
-    "start:dev": "npm run server-versions && npm run style:dev && npm run webpack-dev-server -- --config config/webpack.dev.js --progress --content-base ",
-    "test": "rimraf coverage && karma start --single-run",
+    "ng": "ng",
+    "start": "npm run server-versions && rimraf dist && ng serve --app=0 --open",
+    "start:dev": "npm run server-versions && rimraf dist && ng serve --app=1 pp-dev --proxy-config proxy.conf.json --open",
```

Comment:
maybe it's worth adding --aot to catch problems early?

Reply:
I tried but I get a lot of strange errors when enabled...which I'm not sure why...